packages/ozone-helper/ozone-type/ozone/model/FlowrWarningLog.ts

```diff
@@ -0,0 +1,70 @@
+import { Instant, Item, OzoneType, UUID } from './Item'
+import { isLocationBuilding, LocationBuilding } from './LocationBuilding'
+import { FlowrWarningOperator } from './FlowrWarningOperator'
+import { isLocationSite, LocationSite } from './LocationSite'
+
+export enum LOG_ACTION {
+	LOGIN = 'LOGIN',
+	LOGOUT = 'LOGOUT',
+	SWITCH_SITE = 'SWITCH_SITE',
+	WARNING_ON = 'WARNING_ON',
+	WARNING_OFF = 'WARNING_OFF',
+	WARNING_TEST = 'WARNING_TEST'
+}
+
+@OzoneType('flowr.warning.log')
+export class FlowrWarningLog extends Item {
+
+	date: Instant
+	operator: UUID
+	action: LOG_ACTION
+	targetType?: String
```

Comment:
Why do you need String instead of string ?

Reply:
typo :) good catch

Reply:
There is the same typo in the other type then

Reply:
Yup, copy pasted it ^^